src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -37,6 +37,7 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
         this.addressBook = new AddressBook(addressBook);
         this.userPrefs = new UserPrefs(userPrefs);
         filteredPersons = new FilteredList<>(this.addressBook.getPersonList());
+        filteredAssignments = new FilteredList<>(this.addressBook.getAssignmentList(filteredPersons.get(0)));
```

Comment:
Does this take in account an empty PersonList/ FilteredPersonList?

Reply:
I see, I have on my end modified the getFilteredAssignmentList to return an empty filtered list if the person list is empty due to the ability of the user to delete all the users. I think we will need to have a closer look when we integrate all our respective PRs.